localstack/services/events/events_listener.py

```diff
@@ -138,6 +139,45 @@ def handle_put_rule(data: Dict):
     return True
 
 
+def handle_create_connection(data: Dict, headers: Dict):
+    name = data.get("Name")
+    auth_type = data.get("AuthorizationType")
+    errors = []
+
+    pattern = re.compile("^[\\.\\-_A-Za-z0-9]+$")
+    if not pattern.match(name):
+        error = (
+            "%s at 'name' failed to satisfy: Member must satisfy regular expression pattern: [\\.\\-_A-Za-z0-9]+"
```

Comment:
nit: We're trying to use more f-strings in places like this now  (also applies to the others below)